spec/features/lender_portfolio_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+RSpec.feature "lender portfolio page" do
+  let(:lender) { User.create(email: "example@example.com",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.